src/dotnet/commands/dotnet-migrate/MigrateCommand.cs

```diff
@@ -32,24 +33,48 @@ public MigrateCommand(string templateFile, string projectJson, string sdkVersion
 
         public int Execute()
         {
-            var project = GetProjectJsonPath(_projectJson) ?? GetProjectJsonPath(Directory.GetCurrentDirectory());
-            EnsureNotNull(project, "Unable to find project.json");
-            var projectDirectory = Path.GetDirectoryName(project);
+            var projectsToMigrate = GetProjectsToMigrate(_projectArg);
 
             var msBuildTemplate = _templateFile != null ?
                 ProjectRootElement.TryOpen(_templateFile) : _temporaryDotnetNewProject.MSBuildProject;
 
-            var outputDirectory = projectDirectory;
-
             var sdkVersion = _sdkVersion ?? new ProjectJsonParser(_temporaryDotnetNewProject.ProjectJson).SdkPackageVersion;
             EnsureNotNull(sdkVersion, "Null Sdk Version");
 
-            var migrationSettings = new MigrationSettings(projectDirectory, outputDirectory, sdkVersion, msBuildTemplate, _xprojFilePath);
-            new ProjectMigrator().Migrate(migrationSettings, _skipProjectReferences);
+            foreach (var project in projectsToMigrate)
+            {
+                Console.WriteLine($"START migrating project {project}..");
```

Comment:
At least reduce to one message, "Migrated project {project}" rather than start/end